notebooks/Imputation_best_practices/Imputation_best_practices.ipynb

```diff
@@ -0,0 +1,4557 @@
+{
```

Comment:
<p>Could you please elaborate the prose a bit? Maybe something along these lines:</p><pre class="ql-syntax" spellcheck="false"># Handling Missing Data

One of the first steps in any data science workflow is to understand the dataset and to clean it. This is because real world datasets are often very messy and require significant preprocessing before they can be used for subsequent data science tasks such as feature engineering, model training, etc. One of the tasks within data cleaning is to handle with missing data. There are several approaches that can be taken for missing data, such as dropping it, filling with 0's, filling with mean, interpolating, etc. In this notebook, we will explore some of these techniques, and compare their effectiveness on two sample datasets.

The first sample dataset we will use is random numbers ...

The second sample dataset we will use is UCI housing dataset ...
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I'll add an abstract to explain better. The one you provided looks great and I'll use it with few additions</p>